Icon/Icons/WarningIcon.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import Icon from '../../Icon';
 
-const WarningIcon = ({ color, size }) =>
+const WarningIcon = ({ color, size }) => (
```

Comment:
I think either way `parens` or `no parens` is cool here -- since they both pass lint rules.